lib/round.rb

```diff
@@ -0,0 +1,38 @@
+class Round
+  attr_reader :deck, :turns
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+    @number_correct = 0
+    @number_wrong = 0
+  end
+
+  def current_card
+    @deck.cards.first
+  end
+
+  def take_turn(user_input)
+    turn = Turn.new(user_input, current_card)
+    if turn.correct?
+      @number_correct += 1
+    else
+      @number_wrong += 1
+    end
+    @turns << turn
+    turn
+  end
+
+  def number_correct
+    @number_correct
+  end
```

Comment:
If you had an attr_reader for number_correct, you would not need this method. Either is fine with me!